terragrunt.yaml

```diff
@@ -21,12 +21,10 @@ environment:
 pipeline:
   - uses: git-checkout
     with:
-      expected-commit: dbb03b41976224a2a386970758ffaa78f8dfab7c
+      expected-commit: 705998dc856bd6fbb119db83e9e2bbe681e92ec7
       repository: https://github.com/gruntwork-io/terragrunt
       tag: v${{package.version}}
 
-  - runs: go generate ./...
```

Comment:
It looks like upstream still run `go generate` as part of their build: https://github.com/gruntwork-io/terragrunt/blob/2b1a3e3a9c92de5c058012a5b28381d639caa0d0/Makefile#L53-L54 (pulled in by https://github.com/gruntwork-io/terragrunt/blob/2b1a3e3a9c92de5c058012a5b28381d639caa0d0/Makefile#L33) so I'm not sure we should be removing it here.

Reply:
Thanks for the catch, I really missed it. Looking why it is failing now.